### DISCUSSION TITLE
how do you actually install it on windows

### DISCUSSION THREAD

**Discussion by darkwst** [autobrr/autobrr#825] [Q&A] [answered]
how to install it on windows the only file i see is tar.gz

**Comment by s0up4200** [comment, marked as answer]
> how to install it on windows the only file i see is tar.gz

Hi! Instructions can be found here: https://autobrr.com/installation/windows/

Windows should be able to extract tar.gz files natively in Windows 10 and 11.

**Reply by darkwst** [reply]
so i just add tar.gz to schedule

**Reply by s0up4200** [reply]
You need to extract the exe from the tar.gz file. Then follow the instructions in the link 🙂 

Come by our Discord if you need further help, easier to get help there! https://discord.gg/autobrr
